include/IndexStoreDB/Index/IndexSystem.h

```diff
@@ -15,6 +15,7 @@
 
 #include "IndexStoreDB/Support/LLVM.h"
 #include "IndexStoreDB/Support/Visibility.h"
+#include "indexstore/IndexStoreCXX.h"
```

Comment:
Can just forward declare `indexstore::IndexStoreCreationOptions` rather than including the header.